deploy.md

```
@@ -0,0 +1,118 @@
+	•	
+	•	# Stage 2: Runtime
+	•	FROM python:3.11-slim
+	•	RUN useradd -m appuser && chown -R appuser /app
+	•	WORKDIR /app
+	•	COPY --from=builder /usr/local/lib/python3.11/site-packages /usr/local/lib/python3.11/site-packages
+	•	COPY . .
+	•	RUN chown -R appuser:appuser /app
+	•	USER appuser
+	•	ENV PATH="/home/appuser/.local/bin:$PATH"
+	•	EXPOSE 8000
+	•	HEALTHCHECK --interval=30s --timeout=10s --start-period=5s --retries=3 \
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)